{"uid":"66927ef85763ce7e","name":"test_full_cluster_restart[SIGTERM-postgres]","fullName":"tests.integration.ha_tests.test_self_healing#test_full_cluster_restart","historyId":"1fae0f26d7819f653f6b98b8e1174bd5","time":{"start":1713151514809,"stop":1713151579467,"duration":64658},"description":"This tests checks that a cluster recovers from a full cluster restart.\n\n    The test can be called a full cluster crash when the signal sent to the OS process\n    is SIGKILL.\n    ","descriptionHtml":"<p>This tests checks that a cluster recovers from a full cluster restart.</p>\n<pre><code>The test can be called a full cluster crash when the signal sent to the OS process\nis SIGKILL.\n</code></pre>\n","status":"failed","statusMessage":"AssertionError: Not all units down at the same time.\nassert False","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 340, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 240, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 501, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 119, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 181, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 99, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 849, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 832, in _runtest_for\n    yield\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 883, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 256, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 182, in pytest_runtest_call\n    raise e\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 172, in pytest_runtest_call\n    item.runtest()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1772, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 501, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 119, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 181, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 99, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 195, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 532, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/github-runner/_work/test-runners-self-hosted-postgresql-operator/test-runners-self-hosted-postgresql-operator/tests/integration/ha_tests/test_self_healing.py\", line 275, in test_full_cluster_restart\n    assert await are_all_db_processes_down(\nAssertionError: Not all units down at the same time.\nassert False","flaky":false,"newFailed":true,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"event_loop","time":{"start":1713149844399,"stop":1713149844400,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1713149844400,"stop":1713149844658,"duration":258},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1713149844400,"stop":1713149844400,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1713151506136,"stop":1713151506137,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"reset_restart_condition","time":{"start":1713151506137,"stop":1713151513733,"duration":7596},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"continuous_writes","time":{"start":1713151506137,"stop":1713151506137,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"loop_wait","time":{"start":1713151513734,"stop":1713151514807,"duration":1073},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[{"name":"loop_wait::finalizer","time":{"start":1713151579475,"stop":1713151579514,"duration":39},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"reset_restart_condition::finalizer","time":{"start":1713151579515,"stop":1713151888049,"duration":308534},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"continuous_writes::finalizer","time":{"start":1713151888051,"stop":1713151894127,"duration":6076},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail::0","time":{"start":1713151894129,"stop":1713151894129,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1713153987802,"stop":1713153988191,"duration":389},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1713153988192,"stop":1713153988193,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1713153988193,"stop":1713153988193,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1713153988193,"stop":1713153988193,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.ha_tests"},{"name":"suite","value":"test_self_healing"},{"name":"host","value":"large-415-1ff43b048fda8d52740ef461"},{"name":"thread","value":"12439-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.ha_tests.test_self_healing"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"process","value":"'postgres'"},{"name":"signal","value":"'SIGTERM'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":1,"broken":0,"skipped":0,"passed":3,"unknown":0,"total":4},"items":[{"uid":"3c70be1c5768652b","reportUrl":"../6//#testresult/3c70be1c5768652b","status":"passed","time":{"start":1713059982505,"stop":1713060030026,"duration":47521}},{"uid":"73cb39536a0a0eb8","reportUrl":"../4//#testresult/73cb39536a0a0eb8","status":"passed","time":{"start":1712973132486,"stop":1712973160143,"duration":27657}},{"uid":"82c89df335f12a29","reportUrl":"../2//#testresult/82c89df335f12a29","status":"passed","time":{"start":1712887084876,"stop":1712887111454,"duration":26578}}]},"tags":["asyncio","group(1)"]},"source":"66927ef85763ce7e.json","parameterValues":["'postgres'","'SIGTERM'"]}